### PR TITLE
Makes Skrell genderless

### PR DESCRIPTION
**Pull request by VTCobaltblood**
Skrell will use plural pronouns on examine because they do not care about gender as a concept, and have indistinguishable faces. However, they still have a biological sex, so it's set up in the examine proc and not in the species datums. Genderless underwear todo, but not in this PR.

### CODE REVIEW

**Comment by VTCobaltblood** [comment]
whoa commits are fucked up due to my email change apparently, don't mind it

**Comment by VTCobaltblood** [comment]
!review

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -43,7 +43,7 @@
 	var/list/msg = list("<span class='info'>*---------*\nThis is ")
 
 	var/datum/gender/T = gender_datums[gender]
-	if(skipjumpsuit && skipface) //big suits/masks/helmets make it hard to tell their gender
+	if(skipjumpsuit && skipface || IS_SKRELL) //big suits/masks/helmets make it hard to tell their gender
```
Comment (skull132) [inline code/modules/mob/living/carbon/human/examine.dm]:
```suggestion
	if((skipjumpsuit && skipface) || IS_SKRELL) //big suits/masks/helmets make it hard to tell their gender
```

I'm also uncertain if this is even needed?

Comment (skull132) [reply code/modules/mob/living/carbon/human/examine.dm]: oh no. this is not how you do it. aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.

**Comment by VTCobaltblood** [comment]
I'll reopen this when I come up with a better solution, because there are a lot more gendered phrases in the game than I expected.

**Pull request closed without merging**